### ISSUE TITLE
Lumen 9 exception

### ISSUE THREAD

**Issue by Stajor** [owen-it/laravel-auditing#702]
laravel/lumen-framework 9.0.1
owen-it/laravel-auditing 13.0.2

Exception message
```
In AuditingEventServiceProvider.php line 9:
                                                                                 
Class "Illuminate\Foundation\Support\Providers\EventServiceProvider" not found  
```

**Comment by jenis-daffodil** [comment]
Any updates here? Facing the same issue with Lumen 8.

**Comment by JamesMcClelland** [comment]
The [AuditingEventServiceProvider.php@5](https://github.com/owen-it/laravel-auditing/blob/master/src/AuditingEventServiceProvider.php#L5) file  contains a reference to `Illuminate\Foundation\Support\Providers\EventServiceProvider` which isn't present in later versions of lumen.

However changing the reference to be: `use Laravel\Lumen\Providers\EventServiceProvider as ServiceProvider;` would temporarily resolve it. This could not be applied to the package as it would break the laravel implementation

**Comment by jenis-daffodil** [comment]
My issue has been resolved, after downgrading to "owen-it/laravel-auditing" 12.x

**Comment by MortenDHansen** [comment]
Hi. I am sorry about the missing response. 

We are a bit thin on maintainers for this package in terms of hours. I do not have any experience with Lumen. In order to support usecases including Lumen use, we may need a bit of help.

**Comment by erikn69** [comment]
https://github.com/owen-it/laravel-auditing/pull/731

Try it now

**Comment by parallels999** [comment]
@anteriovieira already fixed by #731, must be closed
